recipes/wavelet_buffer/all/test_package/conanfile.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+from conan import ConanFile
+from conan.tools.build import cross_building
```

Comment:
```suggestion
from conan.tools.build import can_build
```
https://docs.conan.io/en/latest/reference/conanfile/tools/build.html#conan-tools-build-can-run